easybuild/easyconfigs/p/ParallelIO/ParallelIO-2.2.2a-intel-2017a.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'CMakeMake'
+
+name = 'ParallelIO'
+version = '2.2.2a'
+
+homepage = 'https://github.com/NCAR/ParallelIO'
+description = """A high-level Parallel I/O Library for structured grid applications"""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'usempi': True}
+
+# underscore = "${version//./_}"           bash substition like this 
+# sources = ['pio%(underscore)s.tar.gz']    in an easyconfig file ?
```

Comment:
If you want to programmatically change the `sources`, there are a few examples in existing easyconfigs.

e.g. https://github.com/easybuilders/easybuild-easyconfigs/blob/22ab263e7049b39a53a588652d8579107c1255f3/easybuild/easyconfigs/r/ROI_PAC/ROI_PAC-3.0.1-intel-2017a-Perl-5.24.1.eb#L14